src/pages/index.astro

```diff
@@ -43,8 +44,18 @@ await initializeWebContext(Astro, {
     <ElementColors />
     <Metadata entry={getEntry('metadata', 'main')} />
   </Fragment>
-  <Basics entry={getEntry('basics', 'main')} />
-  <Section class="gap-8">
+  <Sidebar>
+    <SidebarItem sectionId="basics" icon="fa6-solid:user">About me</SidebarItem>
```

Comment:
It looks strange, when you scroll to basics, I'd expect to scroll from the top, but looks like that:

![image](https://github.com/user-attachments/assets/161162fc-e600-4c4a-bba6-3771a50c4121)

I understand why, but would be better to scroll to the top of the page like that:
![image](https://github.com/user-attachments/assets/83e71767-2aad-48d5-92e9-6ec80ab42def)

Reply:
Scrolling to the top of the page may be difficult. There is no element to scroll to, and (as visible on the second screenshot) when you're at the top, no section is selected on the sidebar. However, I've added scroll margins to all sections so you can test it again.

Reply:
Thanks for the explanation.
Now looks better than before :)